pyquil/tests/test_quil.py

```diff
@@ -1386,11 +1386,11 @@ def test_params_pi_and_precedence():
     assert _eval_as_np_pi(trivial_pi) == _eval_as_np_pi(exp)
 
     less_trivial_pi = "3 * theta[0] * 2 / (pi)"
-    prog = Program(f"RX({trivial_pi}) 0")
+    prog = Program(f"RX({less_trivial_pi}) 0")
     exp = str(prog[0].params[0])
     assert _eval_as_np_pi(trivial_pi) == _eval_as_np_pi(exp)
```

Comment:
```suggestion
    assert _eval_as_np_pi(less_trivial_pi) == _eval_as_np_pi(exp)
```
??? tests will tell I suppose